addons/website_sale/static/src/js/location_selector/location_selector_dialog/location_selector_dialog.xml

```diff
@@ -1,92 +1,123 @@
+
+                    <!-- List group -->
+                    <div class="d-flex flex-grow-1 h-100 h-md-auto overflow-x-auto">
+                        <div class="list-group list-group-flush flex-grow-1">
+                            <t t-if="this.state.locations.length">
+                                <t t-foreach="this.state.locations" t-as="location" t-key="location.id">
+                                    <Location id="location.id.toString()" name="location.pick_up_point_name" address="location.pick_up_point_address" number="location_index + 1" isSelected="this.state.selectedLocationId === location.id.toString()" setSelectedLocation.bind="setSelectedLocation"/>
                                 </t>
-                                <button
-                                    type="button"
-                                    class="btn btn-primary"
-                                    name="o_location_selector_validate_selection"
-                                    t-att-disabled="!this.state.selectedLocationId"
-                                    t-on-click="validateSelection"
-                                >
-                                    Choose this location
-                                </button>
                             </t>
                             <t t-else="">
-                                SPINNER
+                                <p t-if="this.state.error" class="p-3 fw-bold">No result</p>
+                                <div t-else="" class="position-absolute start-50 top-50 translate-middle">
+                                    <div class="spinner-border" role="status">
+                                        <span class="visually-hidden">Loading...</span>
+                                    </div>
+                                </div>
                             </t>
                         </div>
                     </div>
-                    <div
-                        id="o_location_selector_map_view"
-                        class="tab-pane d-md-flex flex-md-grow-1 justify-content-md-center"
-                        role="tabpanel"
-                    >
-                        <Map
-                            t-if="this.state.loadMap"
-                            locations="this.state.locations"
-                            selectedLocationId="this.state.selectedLocationId.toString()"
-                            setSelectedLocation.bind="setSelectedLocation"
-                        />
-                    </div>
-                </div>
+                </section>
+
+                <!-- Map view -->
+                <section id="o_location_selector_map_view" class="o_locationSelector_view tab-pane fade col-md-8 d-md-flex flex-grow-1 border-start" role="tabpanel" aria-labelledby="list-tab">
+                    <Map t-if="this.state.loadMap" locations="this.state.locations" selectedLocationId="this.state.selectedLocationId.toString()" setSelectedLocation.bind="setSelectedLocation"/>
```

Comment:
Please keep the diff minimal. It will help me review your changes faster. 
```suggestion
                    <Map 
                        t-if="this.state.loadMap" 
                        locations="this.state.locations"
                        selectedLocationId="this.state.selectedLocationId.toString()"
                        setSelectedLocation.bind="setSelectedLocation"
                    />
```

Reply:
Ok, I've made the change 👍